transport/http/handler.go

```diff
@@ -50,6 +51,11 @@ type handler struct {
 func (h handler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	start := time.Now()
 
+	if req.URL != nil && strings.HasPrefix(req.URL.Path, "/debug/") {
```

Comment:
Why are debug pages attached to the http transport? I expected them to be their own thing, available over HTTP/1.1 even when a user did not configure a transport.